### ISSUE TITLE
api.go:655: tabix: name count mismatch:

### ISSUE THREAD

**Issue by sergiigladchuk** [brentp/vcfanno#57]
I have been straggling to annotate my .vcf file with .bed file, which has a lot of HLA and other "variable" chromosomes, like 'chr1_KI270713v1_random'. I was not able to sort this .bed file with sortBed program and index it afterwards, so I used Unix sort commands, and after that I was able to run tabix to get .tbi. Still vcfanno throws en error.

Command:
```
vcfanno test.toml ../vcf/1234567_1_l_r_m_c_lib_g-vardict.vcf > ../../temp/test_annotation.vcf
```
Configuration test.toml file
```
[[annotation]]
file="./sv_manual_sort.bed.gz"
columns=[4]
names=["problematic_region"]
ops=["uniq"]
```
Error:
```
=============================================
vcfanno version 0.2.3 [built with go1.8]

see: https://github.com/brentp/vcfanno
=============================================
vcfanno.go:114: found 1 sources from 1 files
api.go:655: tabix: name count mismatch: 48153 != 3365
```

I have attached .bed and .tbi files
[attachment.zip](https://github.com/brentp/vcfanno/files/883000/attachment.zip)

**Comment by brentp** [comment]
I just made a pull-request to biogo/hts to fix this there. I'll make a new release when that is incorporated.

**Comment by brentp** [comment]
this is fixed in 0.2.6 and above. thanks for reporting.
